pkg/splunk/enterprise/names.go

```diff
@@ -133,6 +145,11 @@ func GetSplunkSmartstoreConfigMapName(identifier string, crKind string) string {
 	return fmt.Sprintf(smartstoreTemplateStr, identifier, strings.ToLower(crKind))
 }
 
+// GetSplunkAppsConfigMapName uses a template to name a Kubernetes ConfigMap for a SplunkEnterprise resource.
+func GetSplunkAppsConfigMapName(identifier string, crKind string) string {
```

Comment:
nit: can we add UT for this?

Reply:
All it is doing is a printf, and it has enough coverage at higher level flows. Let me know if you still prefer a UT

Reply:
I agree, just we have UT for all such APIs, to keep this consistent with them we can have or we can take it later also :)